ruff.toml

```diff
@@ -1,13 +1,28 @@
+line-length = 90
```

Comment:
Why a non-default value?

Reply:
We have several lines that are ~90 characters long that are all comments explaining what the code is doing, particularly in the NumPy session.
I do not feel like editing these to be a few characters shorter as they do a good job as they are and I don't think we need to aggressively try to keep the lines short in this material.